patches/nan/chore_fix_v8_data_internal_field_casting.patch

```diff
@@ -0,0 +1,265 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Mon, 11 Sep 2023 23:39:55 +0200
+Subject: chore: fix v8::Data internal field casting
+
+Changed in https://chromium-review.googlesource.com/c/v8/v8/+/4834471
```

Comment:
> This is a breaking change for embedders
> using v8::Object::GetInternalField() as it changes the return
> type. Since most v8::Value subtypes only support direct casts
> from v8::Value but not v8::Data, calls like
>
> object->GetInternalField(index).As<v8::External>()
>
> needs to be updated to cast the value to v8::Value first:
>
> object->GetInternalField(index).As<v8::Value>().As<v8::External>()


Very good of upstream to detail the commit like this :100:

Reply:
@ckerr it was added by Node.js (`@joyeecheung`) which is probably why!